Dfe.Academies.External.Web/Pages/School/Leases.cshtml.cs

```diff
@@ -9,17 +9,14 @@
 namespace Dfe.Academies.External.Web.Pages.School
 {
 	// TODO:- amend below to us BaseSchoolPageEditModel
-	public class Leases : BasePageEditModel
+	public class Leases : BaseSchoolPageEditModel
 	{
-		private readonly IConversionApplicationCreationService _academisationCreationService;
-		
 		public Leases(IConversionApplicationRetrievalService conversionApplicationRetrievalService,
 			IReferenceDataRetrievalService referenceDataRetrievalService,
 			IConversionApplicationCreationService academisationCreationService) :
 			base(conversionApplicationRetrievalService, 
-				referenceDataRetrievalService)
+				referenceDataRetrievalService, academisationCreationService, "FinancialInvestigations")
 		{
-			_academisationCreationService = academisationCreationService;
 		}
 		
 		[BindProperty]
```

Comment:
sorry @jarednimble , you can get rid of these model properties if you're using the new base class && SchoolName prop